### ISSUE TITLE
Unable to Export maps

### ISSUE THREAD

**Issue by ghost** [lewa-j/Unity-Source-Tools#28]
I am unable to export maps or just do not know how, Any help will be appreciated.

**Comment by Fronkln** [comment]
Are there any errors?

**Comment by ghost** [comment]
Nope, I hit play and it doesn't really export anything even though I have my folder destination set, Unless I'm doing something wrong

**Comment by Fronkln** [comment]
Check console warning tab

**Comment by ghost** [comment]
all that's there is that I'm missing greygrid.vmt and Color_002.vmt

**Comment by Fronkln** [comment]
then the map should load if there are only those warnings and 0 errors

**Comment by ghost** [comment]
I mean it loads..... but how do I export it? That's what I'm asking.

**Comment by Fronkln** [comment]
Model export is very unfinished

**Comment by ghost** [comment]
Oh I'm dumb I forgot to check "Export Map" and when I hit play I got this Error 
NullReferenceException: Object reference not set to an instance of an object
uSrcTools.Test.Load () (at Assets/Code/Test.cs:77)
uSrcTools.Test.Start () (at Assets/Code/Test.cs:62)

**Comment by antim0118** [comment]
yeah, because variable "bsp.map" resets when map is loaded. i forgot to check map exporting because its kinda unusable.
